bigtop-packages/src/rpm/flink/SPECS/flink.spec

```diff
@@ -53,6 +54,14 @@ Source6: bigtop.bom
 Requires: bigtop-utils >= 0.7
 Requires(preun): /sbin/service
 
+%if  %{?suse_version:1}0
```

Comment:
This whole block needs to be removed. Once you add two of the subpackages there's no reason to have main package required these